internals/secrethub/account_delete.go

```diff
@@ -0,0 +1,91 @@
+package secrethub
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/secrethub/secrethub-cli/internals/cli"
+	"github.com/secrethub/secrethub-cli/internals/cli/ui"
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/iterator"
+)
+
+// AccountDeleteCommand is a command to inspect account details.
+type AccountDeleteCommand struct {
+	io        ui.IO
+	newClient newClientFunc
+}
+
+// NewAccountDeleteCommand creates a new AccountDeleteCommand.
+func NewAccountDeleteCommand(io ui.IO, newClient newClientFunc) *AccountDeleteCommand {
+	return &AccountDeleteCommand{
+		io:        io,
+		newClient: newClient,
+	}
+}
+
+// Register registers the command, arguments and flags on the provided Registerer.
+func (cmd *AccountDeleteCommand) Register(r cli.Registerer) {
+	clause := r.Command("delete", "Delete your SecretHub account.")
```

Comment:
We could consider adding a `HelpLong` with information on the prerequisites (e.g. leave/delete orgs) and the impact of the command (permanent deletion of all secrets/repos/dirs etc. without any means of restoring)

Reply:
Added a help long.